docs/user/production-considerations/task-manager-troubleshooting.asciidoc

```diff
@@ -706,3 +706,21 @@ These rough calculations give you a lower bound to the required throughput, whic
 Given these inferred attributes, it would be safe to assume that a single {kib} instance with default settings **would not** provide the required throughput. It is possible that scaling horizontally by adding a couple more {kib} instances will.
 
 For details on scaling Task Manager, see <<task-manager-scaling-guidance>>.
+
+[float]
+[[task-manager-cannot-operate-when-inline-scripts-are-disabled]]
+==== Inline scripts are disabled in {es}
+
+*Problem*:
```

Comment:
Now it feels to me like this section is too low in the doc.  To me, it feels like it should go right about the Diagnosis section.